lithium/lithium.py

```diff
@@ -154,11 +154,9 @@ def processOptions():
     if args.chunksize:
         minimizeMin = args.chunksize
         minimizeMax = args.chunksize
-        repeat = "last"
```

Comment:
You're right, it should set the global variable minimizeRepeat.

Good call, I didn't notice the value changing to "last" from "never". @tysmith, perhaps this was unintentional?

Note that the default value is also defined as "last":

```
grp_add.add_argument(
    stopAfterTime = time.time() + int(a)
    "--repeat",
    default="last",
    choices=["always", "last", "never"],
    help="Whether to repeat a chunk size if chunks are removed. default: last")
```